BLE_GattClient/source/main.cpp

```diff
@@ -463,7 +463,7 @@ class GattClientProcess : private mbed::NonCopyable<GattClientProcess>,
         DiscoveredCharacteristicNode* new_node =
             new(std::nothrow) DiscoveredCharacteristicNode(*characteristic);
 
-        if (new_node == false) {
+        if (new_node == NULL) {
```

Comment:
Why not `nullptr`?

Reply:
good question, my mind keep switching between C++03 and user revision 😅 . Let me change that.

Reply:
I know why I use NULL now. The rest of the file uses NULL so I wanted to keep consistency within the file.